### DISCUSSION TITLE
How can I exclude a page from my navigation while keeping in my collection?

### DISCUSSION THREAD

**Discussion by vdecree** [11ty/eleventy#2810] [Q&A] [answered]
Hi guys,

I want to exclude a page from my navigation but keep it inside my collections.all. In the front-matter how could I target this one page and filter it out? I've tried looking on the docs but I'm struggling to work it out.

This is how I am outputting my nav currently:

```
<nav class="site-header__nav" aria-label="Primary navigation">
        <ul class="[ nav ] [ stack ]" role="list">
        {%- for entry in collections.all | eleventyNavigation %}
          {% if not entry.hide %}
          <li{% if page.url and entry.url in page.url %} class="active"{% endif %}>
              <a href="{% if entry.url %}{{ entry.url }}{% else %}#{% endif %}">
                {{ entry.title }}
              </a>
          </li>
          {% endif %}
        {%- endfor %}
        </ul>
      </nav>
```

**Comment by darthmall** [comment, marked as answer]
It should be the case that any data you add to the `eleventyNavigation` property in your frontmatter is available on the objects returned by `eleventyNavigation`. It looks like you’re trying to use a `hide` property, so you should be able to do

```
---
eleventyNavigation:
  hide: true
---
```

**Comment by vdecree** [comment]
Thank you, worked a treat. I missed this in the docs.

**Comment by galund** [comment]
I'm not seeing this in the docs and it doesn't work for me, ho hum :/

**Reply by darthmall** [reply]
I’m not sure what you’re looking for in the docs, or what you tried that isn’t working, but if you are looking for the `hide` property in the docs, it won’t be there because this isn’t baked into the plugin. This is a solution that OP implemented for themselves by checking for the `hide` property in their template with an if statement:

```html
{% if not entry.hide %}
  <!-- Render the navigation for pages that aren’t hidden -->
{% endif %}
```

And then adding the `hide` property to the `eleventyNavigation` object in frontmatter of pages they want to exclude from rendering in the navigation:

```yaml
eleventyNavigation:
  hide: true
```

**Reply by galund** [reply]
Ah, I see! I'm using some layouts provided by someone else to render these, but now I get it :-) - thank you.

In fact in my case, there doesn't seem to be any bad side-effect from using 
```
---
eleventyExcludeFromCollections: true
---
```

so I've gone with that (just mentioning this in case someone else comes across this discussion :-) )
